style.css

```diff
@@ -0,0 +1,244 @@
+/* mobile first*/
+body {
+  background: #FAFBFF;
+
+}
+
+h1 {
+  font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+  font-style: bold;
+  font-size: 64px;
+  color: #0018A4;
+  margin-top: 64px;
+
+}
+
+.grid-parent {
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: 20px;
+  margin-top: 64px;
+}
+
+/* filter and sorting section */
+h2 {
+  font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+  font-weight: 700;
+  font-style: bold;
+  font-size: 22px;
+  color: black;
+  margin-bottom: 10px;
+}
+
+.filter-and-buttons button {
+  background-color: #CCFFE2;
+  color: #0018A4;
+  border: none;
+  cursor: pointer;
+  border-radius: 50px;
+  padding: 8px 16px 8px 16px;
```

Comment:
`padding: 8px 16px 8px 16px;`
can be shortened to
`padding: 8px 16px;` 
vertical and horisontal values when they are two